src/equations/polytropic_euler_2d.jl

```diff
@@ -0,0 +1,204 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+    @doc raw"""
```

Comment:
I think you need to run the formatter again on this file

Reply:
Done.